lib/resources/example_json/r4_examples_medication_request.rb

```diff
@@ -164,6 +341,3830 @@ module Resources
+                'timing': {
+                  'repeat': {
+                    'boundsPeriod': {
+                      'start': '2020-07-14T08:31:00.000-05:00'
+                    }
+                  },
+                  'code': {
+                    'coding': [
+                      {
+                        'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/4003',
+                        'code': '696617',
+                        'display': 'every week'
+                      },
+                      {
+                        'system': 'http://terminology.hl7.org/CodeSystem/v3-GTSAbbreviation',
+                        'code': 'WK',
+                        'display': 'weekly'
+                      },
+                      {
+                        'system': 'http://snomed.info/sct',
+                        'code': '225769003',
+                        'display': 'Once a week (qualifier value)'
+                      }
+                    ],
+                    'text': 'every week'
+                  }
+                },
+                'route': {
+                  'coding': [
+                    {
+                      'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/4001',
+                      'code': '318190',
+                      'display': 'Topical',
+                      'userSelected': true
+                    },
+                    {
+                      'system': 'http://snomed.info/sct',
+                      'code': '6064005',
+                      'display': 'Topical route (qualifier value)',
+                      'userSelected': false
+                    }
+                  ],
+                  'text': 'Topical'
+                },
+                'doseAndRate': [
+                  {
+                    'doseQuantity': {
+                      'value': 1,
+                      'unit': 'patches',
+                      'system': 'http://unitsofmeasure.org',
+                      'code': '{Patch}'
+                    }
+                  }
+                ]
+              }
+            ],
+            'dispenseRequest': {
+              'validityPeriod': {
+                'start': '2020-07-14T08:31:21.000-05:00'
+              },
+              'numberOfRepeatsAllowed': 0,
+              'quantity': {
+                'value': 12,
+                'unit': 'patches',
+                'system': 'http://unitsofmeasure.org',
+                'code': '{Patch}'
+              }
+            },
+            'substitution': {
+              'allowedBoolean': true
+            },
+            'priorPrescription': {
+              'reference': 'MedicationRequest/313983017'
+            }
+          }
+        },
+        {
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest/314974791',
```

Comment:
These should be closed endpoints
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

Reply:
comment addressed